ScnSoft-Education/LeetCodeTraining.Tests/Longest Palindromic Substring.Tests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using LeetCodeTraining.LongestPalindromicSubstring4;
+using Xunit;
+
+namespace LongestPalindromicSubstringTests
+{
+    public class SolutionTests
+    {
+        [Theory]
+        [InlineData("babad", "bab")]
+        [InlineData("cbbd", "bb")]
+        [InlineData("", "")]
+        [InlineData("aaaaa", "aaaaa")]
+        public void LongestPalindrome_ValidInput_ReturnsExpectedResult(string input, string expectedResult)
+        {
+            // Arrange
+            var solution = new Solution();
+            string result;
+
+            // Act
+            result = solution.LongestPalindrome(input);
```

Comment:
you can define 'result ' variable in one line, No necessity to split it

Reply:
defining the "result" variable in one line